### PR TITLE
added: React Router

### PR DESCRIPTION
**Pull request by roshantrivedi**

### CODE REVIEW

File: React_Day7/avengers/src/components/ErrorComponent.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const ErrorComponent = () => {
+
+    const err = useRouteError();
+
+  return (
+    <>
+        <h2>Sorry mate</h2>
+        <h2>What you're looking for isn't here.</h2>
+        <h4>Why? { err.message || err.statusText} </h4>
+        <a href='/'>Home</a>
```
Comment (nitishnivedan) [inline React_Day7/avengers/src/components/ErrorComponent.js:12]: why anchor tag here.  can better you use Link here or Link in your header component better to always display it for all pages.

File: React_Day7/avengers/src/components/ProfileComponent.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  const { id } = useParams();
+  const [profileInfo, setProfileInfo] = useState([]);
+
+  useEffect(() => {
+    getProfile();
+  }, []);
+
+  async function getProfile() {
+    const data = await fetch(`https://api.github.com/users/${id}`);
+    const d = await data.json();
```
Comment (gavandivya) [inline React_Day7/avengers/src/components/ProfileComponent.js:16]: use some good variable name

File: React_Day7/avengers/src/components/AboutUs.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
```
Comment (gavandivya) [inline React_Day7/avengers/src/components/AboutUs.js:2]: No neeed

**Review by ap221882 — COMMENTED** [review]
Reviewed

**Pull request merged**